vms/platformvm/txs/executor/standard_tx_executor_test.go

```diff
@@ -2024,3 +1983,45 @@ func TestStandardExecutorTransformSubnetTx(t *testing.T) {
 		})
 	}
 }
+
+func defaultTestConfig(t *testing.T, f fork, tm time.Time) *config.Config { //nolint: unparam
+	var (
+		apricotPhase3Time = mockable.MaxTime
+		apricotPhase5Time = mockable.MaxTime
+		banffTime         = mockable.MaxTime
+		cortinaTime       = mockable.MaxTime
+		durangoTime       = mockable.MaxTime
+		eUpgradeTime      = mockable.MaxTime
+	)
+
+	switch f {
+	case eUpgrade:
+		eUpgradeTime = tm
+		fallthrough
+	case durango:
+		durangoTime = tm
+		fallthrough
+	case cortina:
+		cortinaTime = tm
+		fallthrough
+	case banff:
+		banffTime = tm
+		fallthrough
+	case apricotPhase5:
+		apricotPhase5Time = tm
+		fallthrough
+	case apricotPhase3:
+		apricotPhase3Time = tm
+	default:
+		require.FailNow(t, fmt.Sprintf("unhandled fork %d", f))
+	}
+
+	return &config.Config{
+		EUpgradeTime:      eUpgradeTime,
+		DurangoTime:       durangoTime,
+		CortinaTime:       cortinaTime,
+		BanffTime:         banffTime,
+		ApricotPhase5Time: apricotPhase5Time,
+		ApricotPhase3Time: apricotPhase3Time,
```

Comment:
nit: can we re-order this earliest to latest?

Reply:
Done